cmd/assets/bar.json

```
@@ -0,0 +1,183 @@
+		"left": 0,
+		"top": 0,
+        "percent": 100,
+        "center": false,
+		"just_reference_width": true,
+        "data_matrix": "insert",
+        "image_debug": false
+      },
+      {
+        "row_height": 12,
+        "col_width": 9,
+		"top": 1,
```

Biome (2.1.2)

[error] The key top was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)